style.css

```diff
@@ -1,4 +1,150 @@
+/* stylelint-disable color-hex-case */
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
 body {
-    margin: 0;
-    background-color: white;
-  }
+  font-family: "Poppins", sans-serif;
+  background-color: #E5E5E5;
+}
+
+nav {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  position: relative;
+  width: 100%;
+}
+
+nav:hover {
+  box-shadow: 0 0 12px 0 #6070FF;
+}
+
+nav > a {
+  font-size: 18px;
+  color: #6070FF;
+  line-height: 18px;
+  font-weight: 900;
+  text-decoration: none;
+}
+
+.humburger-btn {
+  display: flex;
+  flex-direction: column;
+  width: 40px;
+  height: 30px;
+  position: relative;
+}
+
+.span-1 {
+  position: absolute;
+  width: 100%;
+  height: 6px;
+  background-color: #6070FF;
+  margin-bottom: 3px;
+}
+
+.span-2 {
+  height: 6px;
+  position: absolute;
+  width: 100%;
+  background-color: #6070FF;
+  margin-top: 10px;
+}
+
+.span-3 {
+  height: 6px;
+  position: absolute;
+  width: 100%;
+  background-color: #6070FF;
+  margin-top: 20px;
+}
+
+header {
+  padding: 80px 34px 82px 34px;
+  border-bottom-left-radius: 80px;
+  background-color: #fff;
+  position: relative;
+  background-image: url("./images/header-shapes\ mobile.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+hgroup > h1 {
+  font-size: 30px;
+  font-weight: 700;
+  color: #172B4D;
+  font-family: "Poppins", sans-serif;
+}
+
+hgroup > h2 {
+  color: #172B4D;
+  font-size: 25px;
+}
+
+.Iam {
+  color: #172B4D;
+}
+
+.Iam2 {
+  color: #6070FF;
+  font-weight: 700;
+}
+
+header > p {
+  font-size: 17px;
+  color: #172B4D;
+  padding-top: 30px;
+  margin-bottom: 50px;
+}
+
+.btn {
+  text-decoration: none;
+  background-color: #2230D2;
+  border-color: #2230D2;
+  border-radius: 30px;
+  padding: 10px;
+  font-size: 16px;
+  font-weight: 600;
+  color: white;
+}
+
+.btn:hover {
+  background-color: #6070FF;
+}
```

Comment:
- [ ] Congrats on implementing the hover state for the button. However, you haven't done the active state to match Figma's design.

![image](https://user-images.githubusercontent.com/46839548/153657644-0276f7f2-4380-488f-ad30-bd3b7858491a.png)

If you want to read more about [active](https://developer.mozilla.org/en-US/docs/Web/CSS/:active) pseudo class